src/primitive_target.py

```diff
@@ -10,353 +9,48 @@
 app: vscode
 """
 
-mod.list("symbol_color", desc="Supported symbol colors for token jumping")
-ctx.lists["self.symbol_color"] = {
-    "gray": "default",
-    "blue": "blue",
-    "green": "green",
-    "rose": "red",
-    "squash": "yellow",
-    "plum": "mauve",
-}
-
-
-CONNECTIVES = {"at", "of", "in", "containing"}
-
 BASE_TARGET = {"type": "primitive"}
 STRICT_HERE = {
     "type": "primitive",
     "mark": {"type": "cursor"},
     "selectionType": "token",
     "position": "contents",
     "transformation": {"type": "identity"},
-    "insideOutsideType": "inside",
+    "insideOutsideType": "inside"
 }
 
+parameters = [
```

Comment:
```suggestion
modifiers = [
```

Reply:
Also, I think I might make a capture called `cursorless_modifier` that joins all of these by pipes and then use it below.